test/end-to-end/pages/tasklist/tasklist.js

```diff
@@ -1,24 +1,9 @@
 /* eslint-disable no-await-in-loop */
 'use strict';
-const content = require('app/resources/en/translation/tasklist');
-const testConfig = require('config');
 
 module.exports = async function () {
     const I = this;
-
     await I.checkPageUrl('app/steps/ui/tasklist');
-
-    // we do need to allow refreshing the page here as it takes time to populate ccd, and storing data in the ccd
-    // database gives a success before is actually populated, so is async.
-    for (let i = 0; i <= 5; i++) {
-        const result = await I.waitForText(content.introduction, testConfig.TestWaitForTextToAppear);
-        if (result === true) {
-            break;
-        }
-        await I.refreshPage();
-        await I.wait(3);
-    }
```

Comment:
why would you want to remove this? it seems like there's an async process happening in CCD at this point that this loop can help with